lib/internal/errors.js

```diff
@@ -272,14 +272,13 @@ E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s');
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s');
 E('ERR_UNKNOWN_STDIN_TYPE', 'Unknown stdin file type');
 E('ERR_UNKNOWN_STREAM_TYPE', 'Unknown stream file type');
-E('ERR_VALUE_OUT_OF_RANGE', (start, end, value) => {
-  return `The value of "${start}" must be ${end}. Received "${value}"`;
-});
 E('ERR_V8BREAKITERATOR', 'Full ICU data not installed. ' +
   'See https://github.com/nodejs/node/wiki/Intl');
 E('ERR_VALID_PERFORMANCE_ENTRY_TYPE',
   'At least one valid performance entry type is required');
-E('ERR_VALUE_OUT_OF_RANGE', 'The value of "%s" must be %s. Received "%s"');
+E('ERR_VALUE_OUT_OF_RANGE', (start, end, value) => {
+  return `The value of "${start}" must be ${end}. Received "${value}"`;
```

Comment:
The variable names are misleading. The first parameter is the name of the value and the second is a description of the set of permitted values, so they don't actually depict a `[start, end]` range. Could you change the names while you are at it?